src/pages/registration/files_and_license_tab/FilesTableRow.tsx

```diff
@@ -66,7 +67,25 @@ export const FilesTableRow = ({ file, removeFile, baseFieldName, showFileVersion
   const legalNoteFieldName = `${baseFieldName}.${SpecificFileFieldNames.LegalNote}`;
   const rrsFieldName = `${baseFieldName}.${SpecificFileFieldNames.RightsRetentionStrategy}`;
 
+  const isAcceptedFile = file.publisherAuthority === false;
+
+  const rrsStrategy = file.rightsRetentionStrategy?.configuredType
+    ? file.rightsRetentionStrategy.configuredType
+    : customer?.rightsRetentionStrategy.type;
+
+  const isNullRrs = rrsStrategy === RightsRetentionStrategyTypes.NullRightsRetentionStrategy;
+  const isCustomerRrs = rrsStrategy === RightsRetentionStrategyTypes.RightsRetentionStrategy;
+  const isOverridableRrs = rrsStrategy === RightsRetentionStrategyTypes.OverridableRightsRetentionStrategy;
+
   const fileHasFunderRrs = file.rightsRetentionStrategy?.type === 'FunderRightsRetentionStrategy';
+  const fileHasCustomerRrs = file.rightsRetentionStrategy?.type === 'CustomerRightsRetentionStrategy';
+  const fileHasOverriddenRrs = file.rightsRetentionStrategy?.type === 'OverriddenRightsRetentionStrategy';
```

Comment:
Kommer en endring som gjør at RRS aldri er tom, men vil i stedet ha en ny verdi: `NullRightsRetentionStrategy`
```suggestion
  const fileHasFunderRrs = file.rightsRetentionStrategy.type === 'FunderRightsRetentionStrategy';
  const fileHasCustomerRrs = file.rightsRetentionStrategy.type === 'CustomerRightsRetentionStrategy';
  const fileHasOverriddenRrs = file.rightsRetentionStrategy.type === 'OverriddenRightsRetentionStrategy';
```